src/main/java/com/halotroop/arrchecker/ARRChecker.java

```diff
@@ -36,8 +45,8 @@ public void onInitialize() {
                 } else {
                     String[] validLicenses =
                             {
-                                    "gpl", "mit", "cc0", "apache", "unlicense", "mpl", // Short form names
-                                    "gnu public license", "mozilla public license", "creative commons" // Long form (incorrect, but check anyway)
+                                    "gpl", "lgpl", "mit", "cc0", "apache", "unlicense", "mpl", "wtfpl", // Short form names
```

Comment:
wtfpl is not valid, much to my younger self's dismay. Please remove it.

Reply:
Also, lgpl did not need to be added to the list. If a license contains "gpl" it will mark it as valid.
Adding the long-form version is a good addition though.